README.md

```diff
@@ -33,12 +33,15 @@ This is a test suite for benchmarking various Go serialization methods.
 - [github.com/shamaton/msgpackgen](https://github.com/shamaton/msgpackgen) (generated code)
 - [github.com/mus-format/mus-go](https://github.com/mus-format/mus-go)
 - [wellquite.org/bebop](https://fossil.wellquite.org/bebop) (generated code)
+- [github.com/deneonet/benc](https://github.com/deneonet/benc)
 
 ## Running the benchmarks
 
+To make it fair I use [perflock](https://github.com/aclements/perflock) to lock the CPU usage at 70%
+
 ```bash
 go get -u -t
-go test -bench='.*' ./
+perflock -governor 70% go test -bench='.*' ./ -count=1 > "results.txt"
```

Comment:
Perflock appears to be Linux only, so this this isn't tenable in general. Please re-run the benchmark without it.